src/portals/hope/features/tasks/CheckInMessage.tsx

```diff
@@ -58,22 +60,25 @@ export const CheckInMessage: React.FC = () => {
         </Paragraph>
         <Spacing top="small" />
         <Flex align="center">
-          <Button
-            size="medium"
-            onClick={() => {
-              if (markets.length === 0) {
-                setShowModal(true)
-                return
-              }
+          <HotkeyHint text="check in" keys={[Keys.Option, Keys.Enter]}>
```

Comment:
I think we should start these with a capital letter :)
```suggestion
          <HotkeyHint text="Check in" keys={[Keys.Option, Keys.Enter]}>
```